src/QueryBuilder/DDLQueryBuilderInterface.php

```diff
@@ -424,4 +424,15 @@ public function renameTable(string $oldName, string $newName): string;
      * Note: The method will quote the `table` parameter before using it in the generated SQL.
      */
     public function truncateTable(string $table): string;
+
+    /**
+     * Refresh materialized view
+     *
+     * @param string $viewName The name of the view to refresh.
+     * @param bool $concurrently Refresh the materialized view without locking out concurrent selects on the materialized view.
+     * @param bool|null $withData When `true` then the backing query is executed to provide the new data. Otherwise if `false` then no new data is generated and the materialized view is left in an unscannable state
+     *
+     * @return string The `REFRESH MATERIALIZED VIEW` SQL statement
+     */
+    public function refreshMaterializedView(string $viewName, bool $concurrently = false, ?bool $withData = null): string;
```

Comment:
```suggestion
    public function refreshMaterializedView(string $viewName, bool $concurrently = false): string;
```

Not sure the argument `$withData` is needed. If it is true, it is equal to "truncate" the materialized view. Maybe beter remove it and if the method is required add `truncateMaterializedView()`